src/modules/map/components/LocationPopupCopySlot.vue

```diff
@@ -35,7 +35,7 @@ export default {
             this.buttonText = 'copy_cta'
         },
         async copyValue() {
-            await navigator.clipboard.writeText(this.value)
+            await navigator.clipboard.writeText(this.value || '')
```

Comment:
if default value as described above is implemented, this `||` statement would be obsolete